src/Process/SkipLintProcess.php

```diff
@@ -22,16 +22,10 @@ class SkipLintProcess extends PhpProcess
      */
     public function __construct(PhpExecutable $phpExecutable, array $filesToCheck)
     {
-        $scriptPath = __DIR__ . '/../../bin/skip-linting.php';
-        $script = file_get_contents($scriptPath);
+        $scriptPath = dirname(dirname(__DIR__)) . '/bin/skip-linting.php';
+        $code = sprintf("include('%s');", $scriptPath);
 
-        if (!$script) {
-            throw new RuntimeException("skip-linting.php script not found in '$scriptPath'.");
-        }
-
-        $script = str_replace('<?php', '', $script);
-
-        $parameters = array('-d', 'display_errors=stderr', '-r', $script);
+        $parameters = array('-d', 'display_errors=stderr', '-r', $code);
         parent::__construct($phpExecutable, $parameters, implode(PHP_EOL, $filesToCheck));
```

Comment:
btw: I wonder why this `skip-linting` logic is implemented via subprocess at all. I think parallel-lint would be faster when it just invokes the logic directly.

but thats a story for a future PR.

Reply:
I've been wondering the same, but that was before my time, so will need some history digging (in the old repo) to see if we can figure out the reason. If there is none, I'd welcome a change for this.